packages/opencensus-web-instrumentation-zone/test/test-interaction-tracker.ts

```diff
@@ -399,8 +433,8 @@ describe('InteractionTracker', () => {
         xhr.dispatchEvent(event);
       }, XHR_TIME);
     });
-
-    xhr.open('GET', '/test');
+    if (!urlRequest) urlRequest = '/test';
```

Comment:
You can do this via a default parameter, so `function doHTTPRequest(urlRequest: '/test')`